molgenis-data/src/main/java/org/molgenis/data/system/ImportRun.java

```diff
@@ -1,17 +1,17 @@
 package org.molgenis.data.system;
 
-import com.google.auto.value.AutoValue;
+import java.util.Date;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
 import org.molgenis.data.AttributeMetaData;
 import org.molgenis.data.Entity;
 import org.molgenis.data.meta.system.ImportRunMetaData;
 
-import java.util.Date;
-import java.util.LinkedHashSet;
-import java.util.Set;
 
-@AutoValue
 public class ImportRun extends org.molgenis.data.support.AbstractEntity implements org.molgenis.data.Entity
```

Comment:
What does the ImportRun have to do with the tagwizard fix?
Why replace AutoValue with handwritten hashCode and equals? AutoValue does this for you